docs/content/filejump.md

```diff
@@ -0,0 +1,215 @@
+Choose a number from below, or type in your own value
+[snip]
+XX / FileJump
+   \ "filejump"
+[snip]
+Storage> filejump
+** See help for filejump backend at: https://rclone.org/filejump/ **
+
+You should create an API access token here: https://drive.filejump.com/account-settings
+Enter a string value. Press Enter for the default ("").
+access_token> your_api_token_here
+The ID of the workspace to use. Defaults to personal workspace if not set or 0.
+Enter a string value. Press Enter for the default ("0").
+workspace_id> 0
+Edit advanced config? (y/n)
+y) Yes
+n) No (default)
+y/n> n
+Remote config
+--------------------
+[remote]
+type = filejump
+access_token = your_api_token_here
+workspace_id = 0
+--------------------
+y) Yes this is OK (default)
+e) Edit this remote
+d) Delete this remote
+y/e/d> y
+```
+
+You can then use it like this:
+
+Test with
+
+    rclone lsd remote:
+
+List directories in top level of your FileJump
+
+    rclone ls remote:
+
+List all the files in your FileJump
+
+    rclone copy /home/local/directory remote:directory
+
+Copy `/home/local/directory` to the remote directory called `directory`.
+
+### Getting your API access token
+
+To configure rclone with FileJump you will need to get an API access token.
+
+1. Login to your FileJump account at [https://drive.filejump.com](https://drive.filejump.com)
+2. Go to Account Settings by clicking on your profile in the top right corner
+3. Navigate to the API section
+4. Create a new API access token
+5. Copy the token and paste it into rclone when prompted
+
+### Workspaces
+
+FileJump supports workspaces for organizing your files. By default, rclone will use your personal workspace (workspace_id = 0). If you want to use a different workspace:
+
+1. You can find available workspace IDs in your FileJump account
+2. Set the `workspace_id` in your rclone config to the desired workspace ID
+
+### Root folder ID
+
+You can set the `root_folder_id` to point to a non-root folder.
+
+When you do this, rclone won't be able to see any files in the root folder or any folders above the one you specified.
+
+### Invalid refresh token
+
+If you get an error about an invalid refresh token, this is usually because you have revoked the API access token in your FileJump account settings. You will need to create a new token and update your rclone config.
+
+### Modification times
+
+FileJump doesn't support setting custom modification times via the API. The modification time will be set to the upload time by the server. This means that `--update` and similar flags that rely on modification times may not work as expected.
+
+### Duplicates
+
+FileJump allows duplicate file names in the same directory. When rclone uploads a file, it will first remove any existing files with the same name to avoid duplicates.
+
+### Restricted filenames
+
+FileJump has a minimum filename length requirement of 3 characters. Rclone will automatically pad shorter filenames with spaces to meet this requirement.
```

Comment:
That is the first time I've seen that restriction in a cloud storage system!

I think this needs to be a little bit clearer

> Rclone will automatically pad shorter filenames with spaces on upload to meet this requirement. It will also remove the spaces on download. This means if you happen to have a 1 or 2 character file name ending in a space then syncing may go wrong.